src/Microsoft.AspNetCore.Sockets.Http/HttpSocketOptions.cs

```diff
@@ -18,8 +18,8 @@ public class HttpSocketOptions
 
         public LongPollingOptions LongPolling { get; } = new LongPollingOptions();
 
-        public long PauseWriterThreshold { get; set; } = PipeOptions.Default.PauseWriterThreshold;
+        public long TransportPauseWriterThreshold { get; set; } = PipeOptions.Default.PauseWriterThreshold;
 
-        public long ResumeWriterThreshold { get; set; } = PipeOptions.Default.ResumeWriterThreshold;
+        public long ApplicationPauseWriterThreshold{ get; set; } = PipeOptions.Default.ResumeWriterThreshold;
```

Comment:
Initialize these both to `0` (and initializing this to the default `ResumeWriterThreshold` would be wrong anyway ;)). We'll explore defaults ourselves later.

Reply:
These names aren't great, can we call them TransportMaxBufferSize and ApplicationMaxBufferSize, and do what @anurse said and make them 0.